pkg/config/evaluation.go

```diff
@@ -491,3 +505,15 @@ func checkForNpmScriptsInPackagesV1(npmScript string, config StepConfig, utils p
 	}
 	return false, nil
 }
+
+// anyPreviousStepIsActive loops through previous steps active states and returns true
+// if at least one of them is active, otherwise result is false
+func anyPreviousStepIsActive(runSteps map[string]bool) bool {
```

Comment:
From what I observed, order of previous steps are fixed (by default, it's the one that configured in piper-stage-config.yml file)

Reply:
also, `runSteps` map contains all steps in stage that were configured in this piper-stage-config.yml. So I think, order of previous steps here doesn't matter in this context.

Reply:
@Googlom Maybe we can avoid word 'Previous' in name of the method? 
and instead name it something like anyOtherStepInStageActive or something of that sorts? just to avoid confusion?